test/stream.test.js

```diff
@@ -92,7 +92,7 @@ describe('StreamAPI', () => {
           client.query(q.Update(doc.ref, {}))
         })
         .on('version', (_, event) => {
-          expect(client.getLastTxnTime()).toEqual(event.txnTS)
+          expect(client.getLastTxnTime()).toEqual(event.event.document.ts)
```

Comment:
The transaction time should be tracked against the `event.txn` field.
```suggestion
          expect(client.getLastTxnTime()).toEqual(event.txn)
```

Reply:
@erickpintor That's what I originally thought, but the test wasn't consistently passing 🤔